client/src/components/EditExperiment.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+import { editExperiment } from "../actions/exptActions";
+import { useDispatch } from "react-redux";
+
+
+const EditExperiment = ({ id, name, description, duration, date_started, date_ended, setIsEditing }) => {
+  const dispatch = useDispatch();
+  const [newName, setNewName] = useState(name);
+  const [newDescription, setNewDescription] = useState(description);
```

Comment:
The initial values in `useState()` here are causing errors in the console saying `value prop should not be null` (since some experiments don't have a name or description). I think you should be able to fix it by just adding an empty string as a fallback if it's null. E.g. `useState(name || '')`